components/service/glean/build.gradle

```diff
@@ -2,9 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.25"
+}
+
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+// Configure the Python environments
+envs {
```

Comment:
Even after discussing this with you I admit I'm still a bit confused.

We're moving the setup code from `sdk_generator.gradle` to the library's `build.gradle`. I think that doing this would prevent 3rd party applications to process YAML files: if you create an application external to `android-components`, include *glean* and trigger the build process, it would now probably fail: I'm not sure we'd be able to call the tasks from the *glean*'s gradle.

I claim we should think the sample application as an external entity even though it lives in this repo: it should have no way to access things outside its direct reach, other than through gradle inclusion.

In short, I think it's ok for us to download the environment both in the app and when building the library, and have it in both places.